airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/commands.py

```diff
@@ -58,7 +56,6 @@ async def bump_version(
         ctx.obj["execute_timeout"],
         bump_type,
         changelog_entry,
-        pull_request_number,
```

Comment:
Going "unix style" here: the bump command just bumps version / changelog, it does not open PRs; the pull requests command can be used sequentially following a version bump.